src/sparseml/transformers/utils/helpers.py

```diff
@@ -559,47 +559,53 @@ def fetch_recipe_path(target: str):
     return recipe_path
 
 
-def generate_mask(string: str, prompt: str, censor: str) -> str:
+def generate_mask(string: str, response: str, prompt: Optional[str] = None) -> str:
     """
-    Generate a mask based on provided prompt and censor strings to obscure
-    characters in the input string.
+    Generate a mask based on provided prompt and response strings to obscure
+    characters in the input string. Prompt will be masked and string in response
+    will be kept represented by 0 - remove and 1 - keep.
+    By default, non-reponse wrapped strings will be matched with 0
 
     Args:
     :param string: The input string to be masked.
-    :param prompt: The prompt string to identify characters to keep visible.
-    :param censor: The censor string to identify characters to obscure.
+    :param prompt: The prompt string to identify characters to obscure.
+    :param response: The response string to identify characters to keep visible.
 
     Returns:
         str: A string representing the mask where '1' indicates visible
         characters and '0' indicates obscured characters.
 
     """
+    if prompt is None:
+        prompt = ""
```

Comment:
nit: could skip this and do `prompt: str = ""` in the function definition instead